### ISSUE TITLE
Note overlapping

### ISSUE THREAD

**Issue by versebrik** [ShadowMario/FNF-PsychEngine#2227]
### Describe your bug here.

I'm currently working on moving the mode to a site similar to Scratch.
However, there were cases where two notes were repeatedly overlapped and created. At first, I thought I made it wrong, so I ignored it, but just in case, I deleted the problematic note from the chart editor and rearranged it, and the phenomenon disappeared. The phenomenon started with Kade Engine and was the same with Psych Engine. I'm guessing this problem is caused by copying and pasting the notes to the overlapping location.
In conclusion, when paste a note to an overlapping location, can you add the function of deleting the note from the overlapping location?

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

I think i dont need it.

**Comment by bbpanzu** [comment]
kk

**Comment by tposejank** [comment]
The hell this means

**Comment by bbpanzu** [comment]
when charrting, sometimes copying a section twice will lead to stacked notes and they don't input properly
